arbnode/dataposter/externalsigner/externalsigner.go

```diff
@@ -0,0 +1,92 @@
+package externalsigner
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/signer/core/apitypes"
+	"github.com/holiman/uint256"
+)
+
+type SignTxArgs struct {
+	*apitypes.SendTxArgs
```

Comment:
I was looking to see why the blob fields weren't on `apitypes.SendTxArgs` already, and looking at the comments on `apitypes.SendTxArgs` it says 
```
// This struct is identical to ethapi.TransactionArgs, except for the usage of
// common.MixedcaseAddress in From and To
```
but the fields haven't been copied over yet from `ethapi.TransactionArgs`. This looks like a typical case of code that's supposed to be kept in sync manually being overlooked. For the time being, in `ethapi.TransactionArgs` the fields actually have different types than the types you specified here, so until it's fixed upstream we should probably use the types they already picked.

Reply:
I raised https://github.com/ethereum/go-ethereum/issues/29161 with upstream to see if this is just an oversight or something more.

Reply:
Makes sense, changed to follow types of fields in ethapi.TransactionArgs struct.